garglk/garglk.h

```diff
@@ -117,7 +117,13 @@ typedef struct window_graphics_s window_graphics_t;
 #define SCROLLBACK 512
 #define HISTORYLEN 100
 
+/* limit number of text rows/columns */
+#define MAX_TEXT_COLUMNS 255
+#define MAX_TEXT_ROWS 255
```

Comment:
These are only used in `window.c` so they ought to appear there so as not to pollute the global namespace.

What buffers are overflowed by having > 255 columns/rows?  Is there any reason not to just bump the size of those buffers (which would still require maximums of course), or switch them to dynamic buffers (if they're in C++ code, at least; dealing with dynamic buffers in C is annoying).

Reply:
This change is originally from #323. I don't know if problem it is meant to fix still exists.

Reply:
OK, it looks like the problem exists (or at least _a_ problem with buffer overflows in that area exists).  If the viewport is larger than the scrollback buffer size then there are problems.  Switching to a dynamic scrollback buffer will ultimately be the right way to go but that's a lot of work.  Hard limits like this are fine for now, so just moving these constants into `window.c` works for the time being.

Reply:
Constants moved.